syntax = "proto3";

package com.embedd;

// Import the messages from the other file
import "embedd_messages.proto"; 

service EmbeddService {
    rpc GetEmbedd (EmbeddRequest) returns (EmbeddResponse) {}
}
